browser/src/control/Control.StatusBar.js

```diff
@@ -377,12 +377,19 @@ class StatusBar extends JSDialog.Toolbar {
 			$('#toolbar-down').removeClass('readonly');
 		}
 
+		var permissionContainer = document.getElementById('permissionmode-container');
+		if (permissionContainer) {
+			while (permissionContainer.firstChild)
+				permissionContainer.removeChild(permissionContainer.firstChild);
+			permissionContainer.appendChild(getPermissionModeElements(isReadOnlyMode, this.map['wopi'].UserCanWrite));
+		}
+
 		this.builder.updateWidget(this.parentContainer, {
 			id: 'PermissionMode',
 			type: 'htmlcontent',
 			htmlId: 'permissionmode',
 			isReadOnlyMode: isReadOnlyMode,
-			canUserWrite: !app.isReadOnly()
+			canUserWrite: this.map['wopi'].UserCanWrite
```

Comment:
I wonder if this is ok to do for the android/iOS case? I presume there that this.map['wopi'] would be null, maybe @Minion3665  knows. Would is be safer to use something like:

 this.map['wopi'] ? this.map['wopi'].UserCanWrite : !app.isReadOnly()

Reply:
A bit worse than that I'm afraid - in the mobile case `this.map['wopi']` *isn't* null but `L.Map.THIS.wopi.UserCanWrite` is always false.

here's some stuff from my chrome debug console...
```
$ L.Map.THIS["wopi"]
NewClass {_map: NewClass, _initHooksCalled: true}

$ L.Map.THIS.wopi.UserCanWrite
false
```